packages/spin-operator/v0.2.0+1/package.yaml

```
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "spin-operator"
+shortDescription: "The Spin Operator enables deploying Spin applications to Kubernetes."
+iconUrl: "https://avatars.githubusercontent.com/u/157650797"
+defaultNamespace: "spin-operator"
+helm:
+  repositoryUrl: "oci://ghcr.io/spinkube/charts/spin-operator"
```

flux adds the chart name to the URL, so it would result in "oci://ghcr.io/spinkube/charts/spin-operator/spin-operator" here

```suggestion
  repositoryUrl: "oci://ghcr.io/spinkube/charts"
```